database/src/main/java/database/service/SqlDatabaseService.java

```diff
@@ -0,0 +1,443 @@
+        }
+
+        return entity;
+    }
+
+    @Override
+    public boolean removeRecordFromTable(Class<? extends BaseEntity> entityClass, Integer id) {
+        String tableName = entityClass.getSimpleName();
+        String deleteRecordSQL = "DELETE FROM " + tableName + " WHERE id = " + id;
+
+        LOG.info("Executing SQL: {}", deleteRecordSQL);
+
+        try (Connection connection = DriverManager.getConnection(databaseURL, dbUserName, dbPassword);
+             Statement statement = connection.createStatement()) {
+            statement.executeUpdate(deleteRecordSQL);
+            return true;
+        } catch (SQLException e) {
+            LOG.error(UNABLE_DELETE_RECORD + ": {}, {}", tableName, e.getMessage());
+            throw new DeletionDatabaseException(UNABLE_DELETE_RECORD + ": " + tableName + ", " + e.getMessage());
+        }
+    }
+
+    @Override
+    public void removeAllRecordsFromTable(Class<? extends BaseEntity> entityClass) {
+        String tableName = entityClass.getSimpleName();
+        String deleteAllRecordsSQL = "DELETE FROM " + tableName;
+
+        LOG.info("Executing SQL: {}", deleteAllRecordsSQL);
+
+        try (Connection connection = DriverManager.getConnection(databaseURL, dbUserName, dbPassword);
+             Statement statement = connection.createStatement()) {
+            statement.executeUpdate(deleteAllRecordsSQL);
+        } catch (SQLException e) {
+            LOG.error(UNABLE_DELETE_ALL_RECORDS + ": {}, {}", tableName, e.getMessage());
+            throw new DeletionDatabaseException(UNABLE_DELETE_ALL_RECORDS + ": " + tableName + ", " + e.getMessage());
+        }
+    }
+
+    @Override
+    public <T extends BaseEntity> T getById(Class<? extends BaseEntity> entityClass, Integer id) {
+        String tableName = entityClass.getSimpleName();
+        String selectRecordByIdSQL = "SELECT * FROM " + tableName + " WHERE id = " + id;
+
+        LOG.info("Executing SQL: {}", selectRecordByIdSQL);
+
+        try (Connection connection = DriverManager.getConnection(databaseURL, dbUserName, dbPassword);
+             Statement statement = connection.createStatement();
+             ResultSet resultSet = statement.executeQuery(selectRecordByIdSQL)) {
+            if (resultSet.next()) {
+                return createAndFillEntity(entityClass, resultSet);
+            } else {
+                return null;
+            }
+        } catch (SQLException | ReflectiveOperationException e) {
+            LOG.error(ENTITY_IS_NOT_FOUND + ": {}, {}, {}", id, tableName, e.getMessage());
+            throw new DatabaseOperationException("Error retrieving record by id " + id + ", " + tableName + ", " +
+                    e.getMessage());
+        }
+    }
+
+    @Override
+    public <T extends BaseEntity> Iterable<T> getAllRecordsFromTable(Class<? extends BaseEntity> entityClass) {
+        String tableName = entityClass.getSimpleName();
+        String selectAllRecordsSQL = "SELECT * FROM " + tableName;
```

Comment:
Please use `Limit` value from `Settings`. See how it works in `JsonDatabaseServicegetAllRecordsFromTable`.